File: BBIS.Application/DTOs/Schedule/ChecklistDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BBIS.Application.DTOs.Schedule
{
    public class ChecklistDto
    {
        public int? ChecklistId { get; set; }
        public Guid? ScheduleId { get; set; }
        public string ConfirmationCall { get; set; }
        public string VerificationCall { get; set; }
        public string FinalCall { get; set; }
        public DateTime UpdatedAt { get; set; }
        public Guid UpdatedBy { get; set; }
    }
}

File: BBIS.Api/ClientApp/src/models/Schedules/ChecklistDto.ts
export interface IChecklistDto {
    ChecklistId: number;
    ScheduleId?: string; // Guid -> string
    ConfirmationCall: string;
    VerificationCall: string;
    FinalCall: string;
    UpdatedAt: Date;
    UpdatedBy: string; // Guid -> string
}
export class ChecklistDto implements IChecklistDto {
    ChecklistId: number = 0;
    ScheduleId?: string = "";
    ConfirmationCall: string = "";
    VerificationCall: string = "";
    FinalCall: string = "";
    UpdatedAt: Date = new Date();
    UpdatedBy: string = "";
}